src/tools/sample/sample.c

```diff
@@ -994,6 +1003,90 @@ RunClient(
     }
 }
 
+
+//
+// Runs the multi client side of the protocol.
+//
+void
+RunMultiClient(
+    _In_ int argc,
+    _In_reads_(argc) _Null_terminated_ char* argv[]
+    )
+{
+    //
+    // Load the client configuration based on the "unsecure" command line option.
+    //
+    if (!ClientLoadConfiguration(GetFlag(argc, argv, "unsecure"))) {
+        return;
+    }
+    QUIC_STATUS Status;
+    uint32_t NumberOfConnections = 0;
+    HQUIC* Connections = NULL;
+
+    //
+    // Get the target / server name or IP from the command line.
+    //
+    const char* Target;
+    if ((Target = GetValue(argc, argv, "target")) == NULL) {
```

Comment:
nit: The assignment can be done on the line above, directly when declaring the variable, rather than in the `if` test. Same below.

Reply:
Same pattern seen throughout this file. I want the sample to be consistent, even if it's a pattern that we don't want to use broadly.  We can bulk fix up the sample in a separate PR